docs-chef-io/content/workstation/config_rb.md

```diff
@@ -85,76 +85,76 @@ This configuration file has the following settings:
 `cookbook_path`
 : The Chef Infra Client sub-directory for storing cookbooks. This value can be a string or an array of file system locations, processed in the specified order. The last cookbook is considered to override local modifications. For example:
 
-    ``` ruby
-    cookbook_path [
-      '/var/chef/cookbooks',
-      '/var/chef/site-cookbooks'
-    ]
-    ```
+  ``` ruby
+  cookbook_path [
+    '/var/chef/cookbooks',
+    '/var/chef/site-cookbooks'
+  ]
+  ```
 
 `data_bag_encrypt_version`
 : The minimum required version of data bag encryption. Possible values: `1` or `2`. When all of the machines in an organization are running Chef Client 11.6 (or higher), it is recommended that this value be set to `2`. For example:
 
-    ``` ruby
-    data_bag_encrypt_version 2
-    ```
+  ``` ruby
+  data_bag_encrypt_version 2
+  ```
 
 `fips`
 : Allows OpenSSL to enforce FIPS-validated security during a Chef Infra Client run. Set to `true` to enable FIPS-validated security.
 
 The following operating systems are supported:
```

Comment:
I think this and the following list should be indented to 3 spaces.

```suggestion
   The following operating systems are supported:
```

Reply:
Thanks!